### ISSUE TITLE
Sklearn deprecated

### ISSUE THREAD

**Issue by maesaar** [narning1992/aiSource#2]
Hello @narning1992 

I have tried conda and pip, but they have failed to install `sklearn==0.0`, which is now [deprecated](https://pypi.org/project/sklearn/).  All the other requirements have been installed successfully. I have also used aiSource to predict the source of _C. jejuni_ and _C. coli_ human isolates and no errors have been reported.

My questions:
(1) Is it ok to use aiSource prediction when `sklearn` is not installed?
(2) Can I use aiSource for both _C. jejuni_ and _C. coli_ isolates?

Thank you for the answer

**Comment by maesaar** [comment]
Will post answers I got directly from developer just in case someone is also interested:

> If the program runs through without complain, then no need to worry about the sklearn Installation. If you do want to install it try removing everything after the word sklearn in requirements.txt and try again. That should install the most recent version of sklearn.

> To answer your other question, you can indeed use aisource for coli since they were included in the training data. However the accuracy value are then slightly different. The part should have some information on that.
